docs/source/examples/spot-jobs.rst

```diff
@@ -141,21 +165,23 @@ As HuggingFace has built-in support for periodically checkpointing, we only need
 .. note::
   You may also refer to another example `here <https://github.com/skypilot-org/skypilot/tree/master/examples/spot/resnet_ddp>`_ for periodically checkpointing with PyTorch.
 
-With the above changes, you are ready to launch a spot job with ``sky spot launch``!
+With the above changes, the managed spot job can now resume training after preemption with ``sky spot launch``! We can enjoy the benefits of
+the cost savings from spot instances without worrying about the job being interrupted and losing progress.
```

Comment:
the cost savings -> remove the?

Reply:
worrying about the job being interrupted and losing progress. -> worrying about preemption or losing job progress?

Reply:
Good point! Fixed. Thanks!